proxy/src/main/java/sustain/synopsis/proxy/metadata/ProxyMetadataRequestProcessor.java

```diff
@@ -0,0 +1,78 @@
+package sustain.synopsis.proxy.metadata;
+
+
+import com.google.common.util.concurrent.ListenableFuture;
+import io.grpc.Channel;
+import io.grpc.ManagedChannelBuilder;
+import sustain.synopsis.dht.Context;
+import sustain.synopsis.dht.NodeConfiguration;
+import sustain.synopsis.metadata.*;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+public class ProxyMetadataRequestProcessor implements MetadataRequestProcessor {
```

Comment:
Are we using proxy servers to support multiple metadata servers in a transparent way?